### DISCUSSION TITLE
How to get the X and Y coordinates in the grid component

### DISCUSSION THREAD

**Discussion by julurisaichandu** [pmndrs/drei#1276] [Q&A]
@drcmda I am using a Grid component in react-three/drei and I had to manually add text for each axes for displaying the X and Y coordinates. So is there any way that I can give a range as an input to the Grid component and the range is displayed on the axes as the numbers, instead of adding coordinates to the axes manually?

**Comment by drcmda** [comment]
there's no support for annotations in the code, it's just a grid shader. if you want you want add it in a pr.
